apps/console/src/features/applications/components/settings/attribute-management/attribute-selection.tsx

```diff
@@ -659,7 +659,7 @@ export const AttributeSelection: FunctionComponent<AttributeSelectionPropsInterf
 
     /**
      * Check if the claim has OIDC mapping
-     * @return {boolean}
+     * @returns Is a mapping or not.
```

Comment:
Can we add param to the comment

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/3440/commits/d16981abb35182947ec076270163483cb6bd55f6